x-pack/filebeat/module/gsuite/_meta/fields.yml

```diff
@@ -0,0 +1,40 @@
+- key: gsuite
+  title: "gsuite"
+  description: >
+    gsuite Module
+  fields:
+    - name: gsuite
+      default_field: false
+      type: group
+      description: >
+        Gsuite specific fields.
+
+        More information about specific fields can be found at https://developers.google.com/admin-sdk/reports/v1/reference/activities/list
+      fields:
+        - name: actor.key
+          type: keyword
+          description: >
+            Only present when callerType is KEY. Can be the consumer_key of the requestor for OAuth 2LO API requests or an identifier for robot accounts.
```

Comment:
Wondering if this can be slightly updated with whatever we actually map the field to. It's slightly confusing to me to see a reference to `callerType` when I don't see any reference to it in this `fields.yaml`. If it's unmapped and only in the original payload maybe we can just call that out with something like `Only present when callerType in the original payload is set to "KEY"` or something to that effect?